README.md

```diff
@@ -248,6 +248,15 @@ Jackett can also be run from the command line if you would like to see log messa
        * For other distros see the  [Curl docs](http://curl.haxx.se/dlwiz/?type=devel).
  3. Download and extract the latest `Jackett.Binaries.Mono.tar.gz` release from the [releases page](https://github.com/Jackett/Jackett/releases) and run Jackett using mono with the command `mono --debug JackettConsole.exe`.
 
+## Installation on OSX
+ 1. Install [Mono 4](http://www.mono-project.com/download/#download-mac) or better (version 4.8 is recommended)
+       * Setup ssl support by running
+       ```
+       https://curl.haxx.se/ca/cacert.pem
+       cert-sync --user ~/Downloads/cacert.pem
+       ```
+ 1. Download and extract the latest `Jackett.Binaries.Mono.tar.gz` release from the [releases page](https://github.com/Jackett/Jackett/releases) and run Jackett using mono with the command `mono --debug JackettConsole.exe`.
+
 Detailed instructions for [Ubuntu 14.x](http://www.htpcguides.com/install-jackett-on-ubuntu-14-x-for-custom-torrents-in-sonarr/) and [Ubuntu 15.x](http://www.htpcguides.com/install-jackett-ubuntu-15-x-for-custom-torrents-in-sonarr/)
```

Comment:
I think it's more reasonable to move the section about Ubuntu 14.x and 15.x to the Linux paragraph.